### DISCUSSION TITLE
How to overlay bar plots?

### DISCUSSION THREAD

**Discussion by rongcuid** [has2k1/plotnine#711] [Q&A]
I have a table with a column `x` and a column `total`. I would like to draw both on a bar plot, with `total` at the back and `x` in front (without stacking, of course). I can't seem to find a way to do so. Is it possible?

**Comment by tabedzki** [comment]
Yeah so I had to refer back to the documentation of ggplot to find it out, but you want to do `position="identity"`. That is when I was able to the bars not go over 100%

So in my own code, it looks like 
```python
from mizani.formatters import percent_format

# Using plotnine, generate histogram of the number of clusters per molecule

base = (ggplot(combined_clusters, aes(x="n_clusters",  fill="factor(frac)", y=after_stat('density')) )
    + geom_histogram(binwidth=1, alpha = 0.5, color="black", position="identity")
    + labs(x="Number of clusters per molecule", y="Percentage", title="Histogram of the number of clusters per molecule", fill=r"$f_A$")
    +  scale_y_continuous(labels=percent_format())
 )

print(base)
```
